src/App.js

```diff
@@ -1,13 +1,52 @@
 import React from "react";
+import Heading from "./Components/Heading";
+import Bookings from "./Components/Bookings";
+import Footer from "./Components/Footer";
+import london from "./images/london.jpeg";
+import glasgow from "./images/glasgow.jpeg";
+import manchester from "./images/manchester.jpeg";
+import TouristInfoCards from "./Components/TouristInfoCards";
+import SearchResults from "./Components/SearchResults";
+import Restaurant from "./Restaurant"
 
-import Bookings from "./Bookings";
 import "./App.css";
 
 const App = () => {
+
+  const address = [
+    "123 Fake Street, London, E1 4UD",
+    "hello@fakehotel.com",
+    "0123 456789",
+  ];
+
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <Heading />
+
+      <div className="card-container">
+        <TouristInfoCards
+          image={glasgow}
+          title="Glasgow"
+          text="Glasgow is a port city on the River Clyde in Scotland's western Lowlands. It's famed for its Victorian and art nouveau architecture, Today it's a national cultural hub, home to institutions including the Scottish Opera, Scottish Ballet and National Theatre of Scotland"
```

Comment:
Consider passing as a react fragment to allow you to safely word wrap and not end up with very long lines. It also doesn't add unnecessary elements to the DOM, it's just there for JSX to bundle for you

```
text={
  <>
    Glasgow is a port city...
  </>
}
```